src/templates/directory.tsx

```diff
@@ -18,7 +18,6 @@ import {
   applyAnalytics,
 } from "@yext/visual-editor";
 import { themeConfig } from "../../theme.config";
-import { buildSchema } from "../utils/buildSchema";
 import { AnalyticsProvider } from "@yext/pages-components";
```

Comment:
Hmm, these templates should be moved out and generated by the plugin like main.tsx is.